arviz/plots/forestplot.py

```diff
@@ -158,12 +157,13 @@ def plot_forest(
 
         >>> import arviz as az
         >>> non_centered_data = az.load_arviz_data('non_centered_eight')
-        >>> axes = az.plot_forest(non_centered_data,
+        >>> axes = az.plot_forest_kwargs(non_centered_data,
```

Comment:
The function is still called plot_forest, all occurrences of `plot_forest_kwargs` should be changed back